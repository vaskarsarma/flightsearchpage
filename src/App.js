import Header from "./components/header/header";
import Dashboard from "./components/dashboard/dashboard";
import Footer from "./components/footer/footer";
import "./App.scss";

function App() {
  return (
    <div className="App">
      <Header />
      <Dashboard />
      <Footer />
    </div>
  );
}

export default App;
