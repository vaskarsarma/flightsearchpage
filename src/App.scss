//colors
$color_malibu_approx: #61dafb;

.App {
	text-align: left;
}
.App-logo {
	height: 40vmin;
	pointer-events: none;
}
@media(prefers-reduced-motion: no-preference) {
	.App-logo {
		animation: App-logo-spin infinite 20s linear;
	}
}
.App-link {
	color: $color_malibu_approx;
}
@keyframes App-logo-spin {
	from {
		//Instead of the line below you could use @include transform($scale, $rotate, $transx, $transy, $skewx, $skewy, $originx, $originy)
		transform: rotate(0deg);
	}
	to {
		//Instead of the line below you could use @include transform($scale, $rotate, $transx, $transy, $skewx, $skewy, $originx, $originy)
		transform: rotate(360deg);
	}
}