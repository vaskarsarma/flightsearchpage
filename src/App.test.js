import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import App from "./App";
import Header from "./components/atoms/header/header";
import Dashboard from "./components/organisms/dashboard/dashboard";

describe("<App />", () => {
  let container;

  beforeEach(() => (container = shallow(<App />)));

  it("<APP /> Rendered Correctly", () => {
    expect(shallowToJson(container)).toMatchSnapshot();
  });

  it("should render a <div />", () => {
    expect(container.find("div").length).toEqual(1);
  });

  it("should render the Header Component", () => {
    expect(container.containsMatchingElement(<Header />)).toEqual(true);
  });

  it("should render the Dashboard Component", () => {
    expect(container.containsMatchingElement(<Dashboard />)).toEqual(true);
  });
});
