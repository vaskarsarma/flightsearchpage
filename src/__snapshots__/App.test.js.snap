// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> <APP /> Rendered Correctly 1`] = `
<div
  className="App"
>
  <Header />
  <Dashboard />
  <Footer />
</div>
`;
